dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -899,24 +903,38 @@ STDAPI MrmGetFilePathFromName(_In_ PCWSTR filename, _Outptr_ PWSTR* filePath)
         pointerToPath++;
     }
 
+    pointerToPath = pathAllocated ? pathAllocated.get() : path;
```

Comment:
>pointerToPath = pathAllocated ? pathAllocated.get() : path; [](http://example.com/codeflow?start=4&length=59)

Comment that this resets `pointerToPath` would be helpful.

Reply:
i don't think we want comment that just repeats what the code is doing.

Reply:
ok